### PR TITLE
Add a way for moderators to view user inboxes

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: app/controllers/moderation/inbox_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Moderation::InboxController < ApplicationController
```
Comment (raccube) [inline app/controllers/moderation/inbox_controller.rb:3]: Dio cane.

**Review by nilsding — APPROVED** [review]

File: app/views/moderation/inbox/index.haml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+- provide(:title, generate_title(t(".title", user: @user.screen_name)))
+
+.card.question--fixed
+  .container
+    .card-body
+      .media
+        %a.pull-left{ href: show_user_profile_path(@user.screen_name) }
+          %img.answerbox__question-user-avatar.avatar-md{ src: @user.profile_picture.url(:medium) }
+        .media-body
+          Viewing inbox for
```
Comment (pixeldesu) [inline app/views/moderation/inbox/index.haml]: Locales please!

File: app/views/moderation/inbox/index.haml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+          %br
+          %strong= @user.screen_name
+          = " (##{@user.id})"
```
Comment (pixeldesu) [inline app/views/moderation/inbox/index.haml]: One inbox entry is hidden behind this header.

You need to add duplicated markup to push down the container, see https://github.com/Retrospring/retrospring/blob/main/app/views/question/show.haml#L2-L3

File: app/views/user/_actions.haml [hunk 4]
```diff
@@ -36,6 +36,10 @@
             %a.dropdown-item{ href: '#', data: { target: '#modal-ban', toggle: :modal } }
               %i.fa.fa-ban
               = t 'views.actions.ban'
+        - if current_user.mod? && user != current_user
+          %a.dropdown-item{ href: mod_inbox_index_path(user: user.screen_name) }
+            %i.fa.fa-inbox
+            View inbox
```
Comment (pixeldesu) [inline app/views/user/_actions.haml]: You can add a locale for `views.actions.` in `en.yml` here, I'll migrate it over in a PR when I refactor all user profile translations

**Review by pixeldesu — CHANGES_REQUESTED** [review]

File: app/views/moderation/inbox/index.haml (hunk 2 above)
Comment (raccube) [reply app/views/moderation/inbox/index.haml]: e024d8b

File: app/views/moderation/inbox/index.haml (hunk 3 above)
Comment (raccube) [reply app/views/moderation/inbox/index.haml]: e024d8b

File: app/views/user/_actions.haml (hunk 4 above)
Comment (raccube) [reply app/views/user/_actions.haml]: e024d8b

**Review by pixeldesu — APPROVED** [review]

**Pull request merged**